internal/host/statemachine.go

```diff
@@ -297,26 +299,34 @@ func NewHostStateMachine(th *transitionHandler, log logrus.FieldLogger, disabled
 		PostTransition:   th.PostRefreshHost(statusInfoPreparingForInstallation),
 	})
 
-	// Install host
 	sm.AddTransition(stateswitch.TransitionRule{
```

Comment:
Consider adding a comment for each transition that describe the actual case it covers